runtime/doc/scroll.txt

```diff
@@ -251,4 +251,12 @@ scrolled. This allows you to scroll inactive windows. Note that when scroll
 keys are remapped to keyboard keys, the active window is affected regardless
 of the current cursor position.
 
+							*mouse-scrolling-off*
+If you do not want the mouse to cause scrolling (e.g. because resting your
+palm on the touchpad causes scroll events), you can disable that with: >
+	:map <ScrollWheelDown> <Nop>
+	:map! <ScrollWheelDown> <Nop>
+	:map <ScrollWheelUp> <Nop>
+	:map! <ScrollWheelUp> <Nop>
```

Comment:
Nvim technically suggests `:set mousescroll=hor:0,ver:0` in `:h 'mousescroll'` to disable mouse scrolling, but it seems to still move the cursor. This mapping imo works a bit better as it doesn't do that. :shrug:

Reply:
Does it?

Reply:
And why aren't left and right scrolling keys mapped here?

Reply:
> Does it?

`:h 'mousescroll'` mentions "You can disable mouse scrolling by using a count of 0.". While this disables scrolling, the cursor still seems to move for me (with the mapping it doesn't). :thinking:

Reply:
Funnily enough I can even move it outside of the window's view, and the window doesn't scroll until I move the cursor some other way :thinking:

Reply:
(Should be fixed by #20861)

Reply:
It's gone now, anyway.